packages/lean_analytics/pubspec.yaml

```diff
@@ -0,0 +1,23 @@
+name: lean_analytics
```

Comment:
Up till now our packages included the full company name, so `leancode_analytics`